cmd/homi/docker/service/validator.go

```diff
@@ -159,6 +159,11 @@ var validatorTemplate = `{{ .Name }}:
 {{- end }}
         echo 'ADDITIONAL="$$ADDITIONAL --debug --metrics --prometheus"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
         /klaytn-docker-pkg/bin/k{{ .NodeType }}d start
+{{- if eq .NodeType "cn" }}
+        sleep 1
```

Comment:
1sec is too short to wait open the RPC port.

Reply:
Then, could you please suggest a number for this?

Reply:
For CN, it is enough. Please forget this comment. I resolved this comment. :)